routing/missioncontrol.go

```diff
@@ -299,6 +305,33 @@ func generateBandwidthHints(sourceNode *channeldb.LightningNode,
 	return bandwidthHints, nil
 }
 
+// UpdateEdgePolicy updates edge policy of the routing hints kept in the
+// payment session.
+func (p *paymentSession) UpdateEdgePolicy(e *edgeLocator,
+	update *lnwire.ChannelUpdate) {
+	log.Debugf("Updating edge %v policy in Mission Control", e)
+
+	// Prepare updated values.
+	uMinHTLC := lnwire.MilliSatoshi(update.HtlcMinimumMsat)
+	uFeeRate := lnwire.MilliSatoshi(update.FeeRate)
+	uFeeBase := lnwire.MilliSatoshi(update.BaseFee)
+	uDelta := update.TimeLockDelta
+
+	// Run over hints and update policy if edge id and direction match.
+	for _, edges := range p.additionalEdges {
+		for _, edge := range edges {
+			if edge.ChannelID == e.channelID &&
```

Comment:
I think it could be slightly better to extract the if+update block into a separate function. Saves some indents and temp. vars.

Reply:
Done